utils/build.ps1

```diff
@@ -254,16 +254,47 @@ function Invoke-BuildStep([string]$Name) {
   }
 }
 
-enum Target {
+enum TargetComponent {
   LLVM
   Runtime
   Dispatch
   Foundation
   XCTest
 }
 
-function Get-ProjectBinaryCache($Arch, [Target]$Target) {
-  return "$BinaryCache\" + ($Arch.BuildID + $Target.value__)
+function Get-TargetProjectBinaryCache($Arch, [TargetComponent]$Project) {
+  return "$BinaryCache\" + ($Arch.BuildID + $Project.value__)
```

Comment:
Oh wow, `value__` is the name of the hidden field that .NET generates in structs derived from `System.Enum` per standard. Cool trick!